doc/tutorials/launch_ubuntu.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
How about we put this file with the others in `devicemodel/samples`

Reply:
I am fine with this. @NanlinXie are you fine with this as well?